### ISSUE TITLE
[Question] Module position in full article image in Joomla

### ISSUE THREAD

**Issue by Azot94** [joomla/joomla-cms#13841]
I would like to add module position in full artice image section (instead image) in Joomla! 3.6 what file is responsible for this section?

**Comment by marrouchi** [comment]
Hi, it's not clear what you are trying to achieve here. Can you provide more details ? Or add a screenshot ?

**Comment by Azot94** [comment]
I'll try to explain it more simply. I would like to put the module (or module position) in full artice image section (instead of image) in article viev (above content). But to do that I need to know which files I have to modify and this is what I'm trying to achieve here.

http://i.imgur.com/ImxYuXc.png

**Comment by marrouchi** [comment]
You can acheive this by using the loadmodule plugin : 
https://docs.joomla.org/How_do_you_put_a_module_inside_an_article%3F/fr

Just place the tag {loadposition position[, style]} at the top of your article body and disable the image display.

**Comment by Azot94** [comment]
I thought about it but I have 250 articles and It will be time-consuming...

**Comment by marrouchi** [comment]
You can also  edit your template and add a module position just before the <jdoc:include type="component" />

**Comment by Azot94** [comment]
Thank you for suggestions, but if I still would like to add module directly in full artice image section what file is responsible for this?

`components/com_content/views/article/view.html.php` ?

**Comment by marrouchi** [comment]
Yes.
